source/common/config/delta_subscription_impl.h

```diff
@@ -215,7 +179,57 @@ class DeltaSubscriptionImpl
     stats_.update_attempt_.inc();
   }
 
+  void handleDiscoveryResponse(std::unique_ptr<envoy::api::v2::DeltaDiscoveryResponse>&& message) {
```

Comment:
Can this also be private?

Reply:
It was used in testing, but now it's also part of the new callbacks interface, so it should now feel better to have it public.